modelforge/tests/test_utils.py

```diff
@@ -89,13 +97,15 @@ def test_gaussian_rbf(RBF):
     assert expected_output.shape == (3, n_rbf)
 
 
-@pytest.mark.parametrize("RBF", [GaussianRBF])
+@pytest.mark.parametrize("RBF", [_GaussianRBF])
 def test_rbf_invariance(RBF):
     # Define a set of coordinates
-    coordinates = torch.tensor([[1.0, 2.0, 3.0], [4.0, 5.0, 6.0], [7.0, 8.0, 9.0]])
+    from openmm import unit
```

Comment:
Why are you importing openmm units here? 

We had been using openff units in the rest of the code.  Openff-units are directly compatible with openmm units.  `openff.units.openmm` has  `to_openmm` and  `from_openmm` functions if we do need to convert.